### PR TITLE
Distech

### PR DESCRIPTION
**Pull request by Florie25**

### CODE REVIEW

File: _posts/2021-07-15-Subnero-wins-DisTec.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+banner : images/banner-pulse-standard.jpg
+date:  2021-07-15
+categories: news wnc
+thumbnail: images/pulse-thumbnail-distec.jpeg
+---
+
+We are delighted to announce that we won the [DisTec Undersea Challenge](https://www.udt-global.com/about-the-challenge){:target="_blank"}.
+
+The DisTec Undersea Challenge welcomed innovators and progressive organisations to showcase their solutions and disrupt the undersea defence environment as we know it.
+Our team submitted a 5 min [video](https://www.youtube.com/watch?v=zkaCsabBDoA&t){:target="_blank"} about the Subnero Underwater Network (SUN), which saw us selected for the Grand Finals, yesterday, along side of Marcom Defence and [Seacraft](https://seacraft.eu/){:target="_blank"}.
```
Comment (AnneThKu) [inline _posts/2021-07-15-Subnero-wins-DisTec.md]:
```suggestion
Our team submitted a 5 minute [video](https://www.youtube.com/watch?v=zkaCsabBDoA&t){:target="_blank"} about the Subnero Underwater Network (SUN), which saw us selected for the Grand Finals, yesterday, along side of Marcom Defence and [Seacraft](https://seacraft.eu/){:target="_blank"}.
```

File: _posts/2021-07-15-Subnero-wins-DisTec.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+After a 10 min presentation from all the finalists, we were announced as the winner of the competition.
```
Comment (AnneThKu) [inline _posts/2021-07-15-Subnero-wins-DisTec.md]:
```suggestion
After a 10 minute presentation from all the finalists, we were announced as the winner of the competition.
```

File: _posts/2021-07-15-Subnero-wins-DisTec.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+The Grand Final can be watched [here](https://www.udt-global.com/finalists#:~:text=Watch-,the,-Grand%20Final%20on){:target="_blank"}.
```
Comment (AnneThKu) [inline _posts/2021-07-15-Subnero-wins-DisTec.md]: The link is not working.

File: _posts/2021-07-15-Subnero-wins-DisTec.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+![]({{site.baseurl}}/images/pulse-distec-final-2021.jpeg){: .center-image  }
+
+![]({{site.baseurl}}/images/pulse-distec-finalists-2021.jpeg){: .center-image  }
+
+
+We are now hoping that the travel restrictions, to and from Europe, will be soon lifted so that we can attend the [Undersea Defence Technology - UDT 2021 conference](https://www.udt-global.com/welcome) in Rostock, Germany, in December 2021.
```
Comment (AnneThKu) [inline _posts/2021-07-15-Subnero-wins-DisTec.md:21]: This link is also not working.

**Review by AnneThKu — CHANGES_REQUESTED** [review]

**Comment by AnneThKu** [comment]
Screenshot.
Screenshot 2021-07-21 at 10 08 16 AM

2 more comments:
- Though "Our team submitted a 5 min video.." line is in the new line in the file. It appears in the same paragraph without space after full stop.

- It would be better to have all the text together, so might be good to move "We are now hoping that the travel restrictions, to and from Europe, will be soon lifted so that we can attend the Undersea Defence Technology - UDT 2021 conference in Rostock, Germany, in December 2021." line before the images.

File: _posts/2021-07-15-Subnero-wins-DisTec.md (hunk 3 above)
Comment (Florie25) [reply _posts/2021-07-15-Subnero-wins-DisTec.md]: I'll change the link.

File: _posts/2021-07-15-Subnero-wins-DisTec.md (hunk 4 above)
Comment (Florie25) [reply _posts/2021-07-15-Subnero-wins-DisTec.md:21]: And this one too.

**Review by AnneThKu — APPROVED** [review]

File: _posts/2021-07-15-Subnero-wins-DisTec.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+date:  2021-07-15
+categories: news wnc
+thumbnail: images/pulse-thumbnail-distec.jpeg
+---
+
+We are delighted to announce that we won the [DisTec Undersea Challenge](https://www.udt-global.com/about-the-challenge){:target="_blank"}.
+
+The DisTec Undersea Challenge welcomed innovators and progressive organisations to showcase their solutions and disrupt the undersea defence environment as we know it.
+
+Our team submitted a 5 minutes [video](https://www.youtube.com/watch?v=zkaCsabBDoA&t){:target="_blank"} about the Subnero Underwater Network (SUN), which saw us selected for the Grand Finals, yesterday, along side of Marcom Defence and [Seacraft](https://seacraft.eu/){:target="_blank"}.
```
Comment (manuignatius) [inline _posts/2021-07-15-Subnero-wins-DisTec.md]: Suggest we do not link to the video as that video will be taken down shortly. It is not meant for wider circulation, only to be used till we have a replacement video.

File: _posts/2021-07-15-Subnero-wins-DisTec.md [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+
+After a 10 minutes presentation from all the finalists, we were announced as the winner of the competition.
```
Comment (manuignatius) [inline _posts/2021-07-15-Subnero-wins-DisTec.md]: Judging was done beforehand I think, the presentation was only one of the steps.

File: _posts/2021-05-12-Subnero-Multi-channel-Modems.md [hunk 7]
```diff
@@ -23,11 +23,11 @@ Out-of-the-box, you can use the multi-channel modems as a normal modem or a stan
 
 These devices use spatial diversity combining techniques to decode transmission from remote modems, in real-time. This increases the effective communication range by acting as a spatial diversity receiver.
 
-In addition to the extra receiving channels, the multi-channel modems are also equipped with sensors such as GPS, compass and a depth sensor, for underwater positioning and tracking applications. By combining techniques such as time or phase difference of arrival with the data from these sensors, three dimensional positioning and tracking is also available with the multi-channel modems.
+In addition to the extra receiving channels, the multi-channel modems can also be equipped with sensors such as GPS and compass for underwater positioning and tracking applications. By combining techniques such as time or phase difference of arrival with the data from these sensors, three dimensional positioning and tracking is also available with the multi-channel modems.
```
Comment (manuignatius) [inline _posts/2021-05-12-Subnero-Multi-channel-Modems.md]: Why is this file updated? This should not be part of this PR.

**Review by manuignatius — CHANGES_REQUESTED** [review]

File: _posts/2021-07-15-Subnero-wins-DisTec.md (hunk 5 above)
Comment (Florie25) [reply _posts/2021-07-15-Subnero-wins-DisTec.md]: Ok, I'll amend accordingly.

File: _posts/2021-07-15-Subnero-wins-DisTec.md (hunk 6 above)
Comment (Florie25) [reply _posts/2021-07-15-Subnero-wins-DisTec.md]: We don't need to mention that detail in the Pulse article, in my opinion. The set-up was to give the impression that the judges decided there and then.

File: _posts/2021-05-12-Subnero-Multi-channel-Modems.md (hunk 7 above)
Comment (Florie25) [reply _posts/2021-05-12-Subnero-Multi-channel-Modems.md]: I haven't got a clue what this is doing here. I wasn't there when I requested the review.

Comment (AnneThKu) [reply _posts/2021-05-12-Subnero-Multi-channel-Modems.md]: This is my mistake, when i rebased it to include the fix for pulse link. I will fix it.

File: _posts/2021-07-15-Subnero-wins-DisTec.md (hunk 6 above)
Comment (manuignatius) [reply _posts/2021-07-15-Subnero-wins-DisTec.md]: I felt it gives the impression that we won after 10 mins - sounds like an easy win.  The proposal was to just remove the 10mins part, and go with "after the presentation from..."

Comment (manuignatius) [reply _posts/2021-07-15-Subnero-wins-DisTec.md]: Go ahead, if you think its not a big deal.

Comment (Florie25) [reply _posts/2021-07-15-Subnero-wins-DisTec.md]: I'm with you now, I'll amend accordingly.

**Review by manuignatius — APPROVED** [review]
